package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -314,13 +494,42 @@ class WaterBridgeAnalysis(HydrogenBondAnalysis):
 
     .. versionadded:: 0.17.0
```

Comment:
You need a `.. versionchanged 0.20.0` to note the changes that a user needs to know about

Reply:
version change added.